lms/envs/aws.py

```diff
@@ -470,6 +470,10 @@
 ##### ADVANCED_SECURITY_CONFIG #####
 ADVANCED_SECURITY_CONFIG = ENV_TOKENS.get('ADVANCED_SECURITY_CONFIG', {})
 
+##### SHIB #####
+SHIB_ONLY_SITE = ENV_TOKENS.get('SHIB_ONLY_SITE', SHIB_ONLY_SITE)
+SHIB_REDIRECT_DOMAIN_WHITELIST = ENV_TOKENS.get('SHIB_REDIRECT_DOMAIN_WHITELIST', {})
```

Comment:
Should this default to `SHIB_REDIRECT_DOMAIN_WHITELIST` imported from `common.py`?